packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -24,7 +24,7 @@ export async function openParentRevHub(serialNumber: string): Promise<ExpansionH
 
     let serial = openSerialMap.get(serialPortPath)!;
 
-    let parentHub = new ExpansionHubInternal();
+    let parentHub = new ExpansionHubInternal(true, serial);
```

Comment:
When we merge in the children PR and this one, should children receive their parents' serial, or should the serial be undefined for children?

Reply:
The serial should be undefined for children (and not exposed at all in `RevHub` or `ExpansionHub`, only `ParentRevHub`).